project/build.scala

```diff
@@ -69,7 +70,8 @@ object ScalatraBuild extends Build {
     base = file("core"),
     settings = scalatraSettings ++ Seq(
       libraryDependencies <++= scalaVersion(sv => {
-        val defau = Seq(servletApi % "provided;test",
+        val default = Seq(servletApi % "provided;test",
+          reflect(sv),
```

Comment:
scala-reflect is a required dependency for both 2.10 and 2.11. Take a look here:

- http://docs.scala-lang.org/overviews/quasiquotes/setup.html
- https://github.com/scalamacros/sbt-example-paradise/blob/master/project/Build.scala